gravitee-apim-console-webui/src/management/api/portal/ng-subscriptions/components/dialogs/creation/api-portal-subscription-creation-dialog.component.ts

```diff
@@ -188,7 +191,7 @@ export class ApiPortalSubscriptionCreationDialogComponent implements OnInit, OnD
     return application?.name;
   }
 
-  shouldDisplayCustomApiKey(): boolean {
+  private shouldDisplayCustomApiKey(): boolean {
```

Comment:
Is it still use ?
Shouldn't we also add
``` 
this.form.get('selectedPlan').value?.apiId && this.form.get('selectedApplication').value?.id
```
?

Reply:
It's only used in the `dialogResult` that's calculated with the method `onCreate`. I think I can simplify it so that the method `shouldDisplayCustomApiKey` is no longer necessary.